src/help/zaphelp/contents/ui/tabs/spider.html

```diff
@@ -48,8 +48,26 @@ <h1>Spider tab</h1>
 	</ul>
 
 	<p>For each spider message, shown under the Messages tab, you can see details of the request sent and response
-		received, moreover it's indicated, in the column <code>Processed</code>, whether or not the message was processed
-		(that is, the response was successfully received and parsed).
+		received, moreover it's indicated, in the column <code>Processed</code>, whether or not the message was processed:
```

Comment:
I'm going to suggest a slight change here:
`received. The <code>Processed</code> column, indicates whether...`